javaslang/generator/Generator.scala

```diff
@@ -1558,6 +1569,15 @@ def generateTestClasses(): Unit = {
         """
       }
 
+      def genSimpleAliasTest(name: String, value: String): String = {
+        xs"""
+          @$test
+          public void should${name}ReturnNotNull() {
+              assertThat(${name.toLowerCase}($value)).isNotNull();
```

Comment:
If name is denoted in camel case, we should only convert the first character to lower case.